src/msg/async/AsyncMessenger.cc

```diff
@@ -202,6 +208,12 @@ void Processor::accept()
 	} else {
 	  lderr(msgr->cct) << __func__ << " no incoming connection?"
 			   << " errno " << r << " " << cpp_strerror(r) << dendl;
+          if (++accept_error_num > msgr->cct->_conf->ms_accept_error_nums) {
```

Comment:
i think it'd better to just fail the process if we are sure we are facing an unrecoverable error, i.e. call `ceph_abort()` here.

Reply:
Yes, ceph_abort() works and it's simple.